webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/ComplianceData.scala

```diff
@@ -526,14 +526,14 @@ object ComplianceData extends Loggable {
 
     //we can have rules with only overriden reports, so we just prepend them. When
     //a rule is defined for that id, it will override that default.
-    val overridesRules = overridesByRules.view.mapValues(_ => AggregatedStatusReport(Nil))
+    val overridesRules = overridesByRules.view.mapValues(_ => AggregatedStatusReport(Nil)).toMap
```

Comment:
This is important, it was the cause of directive duplication in node's compliance screen

Reply:
Why keeping the view if there's a toMap right after?